### PR TITLE
git-release: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5137 

This PR and command is cursed, it has coursed me nothing but trouble, I accidentally made a tag on tldr rather than my test repo. 
I had to rebase for the first time because of a commit between a good one and another good one

### CODE REVIEW

**Review by bl-ue — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
👍🏽

File: pages/common/git-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# git release
+
+> Create a Git release tag.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-release>.
+
+- Create a Git release, locally and remotely:
```
Comment (SethFalco) [inline pages/common/git-release.md:7]: What do you think of this? It's also a bit more accurate.

```suggestion
- Create and push a Git release:
```

**Review by SethFalco — COMMENTED** [review]
I think the page looks great, just had one subjective suggestion.

Also, just a question, the page refers to "Git releases", but is a "Release" a native Git feature? I thought Git just has tagging, and that's utilized by GitHub/GitLab/etc for releases?

File: pages/common/git-release.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+> Create a Git release tag.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-release>.
+
+- Create and push a Git release:
+
+`git release {{tag_name}}`
+
+- Create a signed Git release, locally and remotely:
```
Comment (CleanMachine1) [inline pages/common/git-release.md:11]:
```suggestion
- Create and push a signed Git release:
```

File: pages/common/git-release.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+`git release {{tag_name}} -s`
+
+- Create a Git release with a message, locally and remotely:
```
Comment (CleanMachine1) [inline pages/common/git-release.md:15]:
```suggestion
- Create and push a Git release with a message:
```

**Comment by SethFalco** [comment]
You sure it was fine to merge this already? I gave an additional comment in https://github.com/tldr-pages/tldr/pull/6095#pullrequestreview-676807039
Do you have an opinion on that?

**Comment by CleanMachine1** [comment]
...
Mistakes have been made. However it is making a type of tag release thing.

It isn't that big of a deal regardless

**Comment by SethFalco** [comment]
I'll leave that for others to decide. To me, it's a big deal as someone who learns a lot from inference.
If I wasn't familiar with Git, I would've wrongfully learned that "Releases" are a Git feature.

While I am on an extreme when it comes to taking things literally, it's also a more accessible as this is common in people with Autism for example.

> taking things very literally – for example, you may not understand sarcasm or phrases like "break a leg"
> \- https://www.nhs.uk/conditions/autism/signs/adults/

**Comment by CleanMachine1** [comment]
Well if you would like me to revert the PR, I am perfectly willing to :+1:

**Comment by CleanMachine1** [comment]
I'll make an entire PR instead

**Pull request merged**